site/_layouts/post.html

```diff
@@ -0,0 +1,80 @@
+<!-- Blog Layout Start -->
+{% assign abc = page.author | split: "," %}
+{% if abc.size > 1 %}
+{% assign img_link = abc.last %}
+{% else %}
+{% assign img_link = "http://pageanthive.com/images/empty-profile.png" %}
+{% endif %}
+{% assign name = abc.first %}
+<!DOCTYPE html>
+<html lang="{{ site.lang }}">
+
+  {% include head.html %}
+
+  <body id="page-top" data-spy="scroll" data-target=".navbar"  data-offset="151">
+
+    {% include navigation.html %}
+    {% if page.title == 'Blog' %}
+    <div class="container-fluid" id="topCont">
+      <div class="row">
+        <img src="{{ 'assets/images/blogTop.png' | relative_url }}">
+      </div>
+    </div>
+    {% endif %}
+
+    <div class="container" id="postsContainer">
+      <div class="row">
+    {% if page.title != 'Blog' %}
+      <div class="col-md-10 col-lg-10 col-md-offset-1 col-lg-offset-1">
+        <div class="row blogPost">
+          <div class="col-md-12 col-lg-12">
+            <div class="row postTop">
+              <p class="timeStamp">{{ name }} blogged on {{ page.created_at | date: "%a, %d %b %Y" }}</p>
+              <img class="postWriter" src="{{ img_link }}" style="min-width:150px; min-height:150px; max-height:150px; max-width:150px;">
+              <p class="postTitle">{{ page.title }}</p>
+            </div>
+            <hr>
+            <div class="row postContent">
+              <p>
+                {{ page.content }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    {% else %}
+      {% if site.posts.size == 0 %}
+        <h2>No blogs have been added until now. Stay tuned !</h2>
+      {% endif %}
+      {% for post in site.posts %}
+        <div class="col-md-10 col-lg-10 col-md-offset-1 col-lg-offset-1">
+          <div class="row blogPost">
+            <div class="col-md-12 col-lg-12">
+              <div class="row postTop">
+                {% assign authp = post.author | split: "," %}
+                <p class="timeStamp">{{ authp.first }} blogged on {{ post.created_at | date: "%a, %d %b %Y" }}</p>
+                <img class="postWriter" src="{{ authp[1] | default: 'http://pageanthive.com/images/empty-profile.png' }}" style="min-width:150px; min-height:150px; max-height:150px; max-width:150px;">
+                <p class="postTitle"><a style="color:inherit; text-decoration:none" href="{{post.url | relative_url}}">{{ post.title }}</a></p>
+              </div>
+              <hr>
+              <div class="row postContent">
+                <p>
+                  {{ post.content | strip_html | truncate: 500 }}
```

Comment:
What's the nature of feeds? 

I thought that they had a separate summary and content fields.

If this is the case, we should be using summary here.

In any case, we should document this in a "usage guide" somewhere for people adding their own blogs. I guess wordpress has this automatically handled, but for instance if someone uses `jekyll`, best to inform them what their feed must contain so that their blog can be integrated here.

Reply:
The issue with summary is that it can be arbitrary html in feeds, I don't want that arbitrary html to be messing up with our webpage, instead I take down the content and strip html myself

Although probably we can try using content, but again this is limited by the scope of the current gem, we would need to either update the gem at upstream or modify it and use it from our forks/ include it as a plugin in this repo itself

Reply:
Btw, there's a nice picture summarising what all we can have in a feed:
https://github.com/feedreader/pluto/tree/master/pluto-models

Reply:
> The issue with summary is that it can be arbitrary html in feeds, I don't want that arbitrary html to be messing up with our webpage, instead I take down the content and strip html myself

But `post.content` can also have arbitary html? That's why you're using `strip_html` right?

What I was suggesting was do `{{ post.summary | strip_html | truncate: 500 }}` instead of `{{post.content | strip_html | truncate: 500 }}`

Though as you said, it may not be available in this gem